sdk/identity/azure-identity/azure/identity/_credentials/cli_credential.py

```diff
@@ -0,0 +1,66 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+import sys
+_IS_PY2 = sys.version_info[0] == 2
```

Comment:
Can you see if we have this "is py2" check/variable in Azure core?  I have an inkling that it is already available.  The same goes for "is windows" check.

Reply:
I don't find either one as a utility method/variable.

Reply:
This _IS_PY2 is removed though, I'm now using six instead.